internal/acceptance/openstack/compute/v2/aggregates_test.go

```diff
@@ -46,9 +46,11 @@ func TestAggregatesCRUD(t *testing.T) {
 
 	tools.PrintResource(t, aggregate)
 
+	name := "new_aggregate_name"
+	availabilityZone := "new_azone"
 	updateOpts := aggregates.UpdateOpts{
-		Name:             "new_aggregate_name",
-		AvailabilityZone: "new_azone",
+		Name:             &name,
+		AvailabilityZone: &availabilityZone,
```

Comment:
why internal? this can be reused in other packages. also it would be helpful to have a helper to get a pointer value on-the-fly. and if it's a nil, return an empty struct.

Reply:
Internal so that it doesn't increase the package's API. I don't think we want that function to be part of the exported Gophercloud funtions.

>it would be helpful to have a helper to get a pointer value on-the-fly. and if it's a nil, return an empty struct

That looks complex. Do you envision that as another convenience function for testing? Would you use that in this PR?